.gitignore

```diff
@@ -11,3 +11,5 @@ pom.xml
 *#
 .#*
 */.lein-repl-history
+*/out/
+.nrepl-port
```

Comment:
Created a new Pull request for the path thing. Please see https://github.com/lshift/cloverage/pull/70